.github/workflows/test.yml

```diff
@@ -280,50 +246,51 @@ jobs:
       - name: Download node image
         uses: actions/download-artifact@v2
         with:
-          name: cheqd-cli-image.tar
+          name: cheqd-node-image.tar
 
       - name: Load node image
-        run: docker load -i cheqd-cli-image.tar
+        run: docker load -i cheqd-node-image.tar
 
-      - uses: actions/checkout@v2
+      - name: Check out
+        uses: actions/checkout@v2
 
-      - name: Set up 4 validators + 2 observers node docker pool for local test run # Replace with testnet image
-        working-directory: ./tests/networks/docker-compose-localnet
+      - name: Set up 4 validators + 2 observers node docker pool
+        working-directory: ./docker/localnet
         run: |
           bash gen-network-config.sh
-          docker-compose up -d
+          CHEQD_NODE_IMAGE=cheqd-node CHEQD_NODE_VERSION=latest docker compose up -d
```

Comment:
If you want to call/reference the file docker-compose.env for any variables you need in Docker Compose file itself (as opposed to within the container), the correct syntax for this is `docker compose --env-file <filepath to docker-compose.env> up -d`. You don't need to do it inline like this. 

BTW, this is what I meant by the fact that the `env_file` in a Docker Compose file is entirely different than the `env-file` flag for Docker Compose CLI. They don't make it easy!

Reply:
Env file isn't supported by `docker compose` (with space) subcommand, `docker-compose` standalone utility is outdated on GitHub runners so I did that. So I see options:
* Update `docker-compose` manually and use it
Oh, I see only one option :)

Reply:
Hmmm it's worth checking if the Docker-compose failure is because GitHub doesn't support it or if the Actions fails due to a similar reason. https://github.community/t/how-to-use-docker-compose-with-github-actions/16850/2

I do agree about needing to override certain commands. I guess the question is...the test pipeline tests against a localnet, correct? The persistent chain image can have a fixed version number, pulled from GHCR but for the test pipeline, I would image the version number and so can be declared as environment variables in the command. Basically there's a lot besides the node moniker and node version that is declared in the env file, that might be useful to actually take from the env file. What do you think?

Reply:
> Hmmm it's worth checking if the Docker-compose failure is because GitHub doesn't support it or if the Actions fails due to a similar reason. https://github.community/t/how-to-use-docker-compose-with-github-actions/16850/2

We could use `--env-file` but we have to override all values in it: image name and image version. So imo there is no need for `--env-file`. Would you agree?

> I do agree about needing to override certain commands. I guess the question is...the test pipeline tests against a localnet, correct?

Yes

> ...but for the test pipeline, I would image the version number and so can be declared as environment variables in the command.

Here they are:

```
CHEQD_NODE_IMAGE=cheqd-node CHEQD_NODE_VERSION=latest docker compose up -d
```

There is no flag like `--env` in compose.

> Basically there's a lot besides the node moniker and node version that is declared in the env file, that might be useful to actually take from the env file. What do you think?

Node monikers are container-related so they are in container env files like [this one](https://github.com/cheqd/cheqd-node/pull/280/files/b743a7aa870b3c981f5864b1ebd3965de0ffff67#diff-730343d940a9be6d13468423951c9d5e00514286773a0ce5b1304d5342c40f45).

On the other hand image name and version are compose related parameters so they are in [docker-compose.env](https://github.com/cheqd/cheqd-node/pull/280/files/b743a7aa870b3c981f5864b1ebd3965de0ffff67#diff-3d4036a5f8ce5d689d2fb94ab18543fb255922399483d303b69968388c5de7e2).

In the current scenario we override both image and version so there is no need for `--env-file` flag in my opinion.